invenio_rdm_records/config.py

```diff
@@ -559,6 +559,19 @@ def lock_edit_published_files(service, identity, record=None, draft=None):
 RDM_RECORDS_MAX_MEDIA_FILES_COUNT = 100
 """Max amount of media files allowed to upload in the deposit form."""
 
+RDM_MEDIA_FILES_DEFAULT_QUOTA_SIZE = 10**10
+"""Default size for a bucket in bytes for media files."""
+
+RDM_MEDIA_FILES_DEFAULT_MAX_FILE_SIZE = RDM_MEDIA_FILES_DEFAULT_QUOTA_SIZE
```

Comment:
Let's set them both explicitly instead of referencing each other because the configuration variables are not that smart and are loaded explicitly, eg. from env, invenio.cfg.